docs/core_systems/service_orchestration_dynamic.md

```diff
@@ -1,9 +1,61 @@
 # Service Orchestration (Dynamic)
 
-Coming soon.
+This core system exists to find matching service instances according to the consumer’s specification within an Eclipse Arrowhead Local Cloud (LC) and optionally, in other Arrowhead clouds by collaborating with other core/support Systems. 
+
+Learn more: <br />
+:material-file-document: [Abstract System Description (SysD)](../assets/sysd/5_0_0/serviceorchestration_sysd.pdf)
 
 ## Services
 
+### orchestration
+
+The purpose of this service is to get matching service instances. The service is offered for both application and core/support systems.
+
+Learn more: <br />
+:material-file-document: [Abstract Service Description (SD)](../assets/sd/5_0_0/orchestration_sd.pdf) <br />
+:material-api: [generic-http (IDD)](../api/serviceorchestration/orchestration-generic-http.md) | [generic-https (IDD)](../api/serviceorchestration/orchestration-generic-http.md) <br />
+:material-api: [generic-mqtt (IDD)](todo) | [generic-mqtts (IDD)](todo) <br />
+:material-tag: since: v5.0.0 
+
+**pull**
+
+This service operation performs the orchestration process and returns the matching service instances.
+
+:material-arrow-right-thin: Example: [generic-http](../api/serviceorchestration/orchestration-generic-http.md#pull) | [generic-https](../api/serviceorchestration/orchestration-generic-http.md#pull)<br />
+:material-arrow-right-thin: Example: [generic-mqtt](todo) | [generic-mqtts](todo)
+
+**subscribe**
+
+This service operation creates a subscription that can be triggered anytime to perform the orchestration process and push the matching service instances for the subscriptor.
+
+:material-arrow-right-thin: Example: [generic-http](../api/serviceorchestration/orchestration-generic-http.md#subscribe) | [generic-https](../api/serviceorchestration/orchestration-generic-http.md#subscribe)<br />
+:material-arrow-right-thin: Example: [generic-mqtt](todo) | [generic-mqtts](todo)
+
+**unsubscribe**
+
+This service operation removes a subscription.
+
+:material-arrow-right-thin: Example: [generic-http](../api/serviceorchestration/orchestration-generic-http.md#unsubscribe) | [generic-https](../api/serviceorchestration/orchestration-generic-http.md#unsubscribe)<br />
+:material-arrow-right-thin: Example: [generic-mqtt](todo) | [generic-mqtts](todo)
+
+### orchestration-push-management
+
+Coming soon.
+
+### simple-store-management
+
+Coming soon.
+
+### flexible-store-management
+
+Coming soon.
+
+### orchestration-lock-management
+
+Coming soon.
+
+### orchestration-history-management
+
```

Comment:
The links in chagelog are wrong: should be similar to this `../general/changelogs/cl500.md#general`